### PR TITLE
Add crypto and Schedule B Part III questions

### PR DESCRIPTION
**Pull request by zakpatterson**
I added a new Questions component that tries to calculate the minimal set of questions that will need to be answered based on the current state. There is only one question now that always needs to be answered. Next up would be to support the questions from schedule B (#118).

### CODE REVIEW

File: src/data/questions.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export interface Question {
+  text: string
+  required: Either<boolean, (state: TaxesState) => boolean>
+  tag: QuestionTag
+}
+
+// function q (tag: QuestionTag, text: string, required: ((s: TaxesState) => boolean)): Question {
+//   return { text, tag, required: right(required) }
+// }
```
Comment (thegrims) [inline src/data/questions.ts:18]: Are the function and type here intended to be commented out?

File: src/irsForms/ScheduleB.ts [hunk 2]
```diff
@@ -70,14 +70,19 @@ export default class ScheduleB implements Form {
     sumFields(this.l5Fields().map(({ amount }) => amount))
   )
 
+  foreignAccount = (): boolean => this.state.questions.FOREIGN_ACCOUNT_EXISTS ?? false
+  fincenForm = (): boolean => this.state.questions.FINCEN_114 ?? false
+  fincenCountry = (): string | undefined => this.state.questions.FINCEN_114_ACCOUNT_COUNTRY
+  foreignTrust = (): boolean => this.state.questions.FOREIGN_TRUST_RELATIONSHIP ?? false
+
   // TODO - FINCEN questions
```
Comment (thegrims) [inline src/irsForms/ScheduleB.ts:78]: Are FINCEN questions still TODO?

**Comment by thegrims** [comment]
Noticed this warning while running the questions tests 

`FOREIGN_TRUST_RELATIONSHIP is missing in the 'defaultValue' prop of either its Controller (https://react-hook-form.com/api#Controller) or useForm (https://react-hook-form.com/api#useForm)`

**Comment by zakpatterson** [comment]
> Noticed this warning while running the questions tests
> 
> `FOREIGN_TRUST_RELATIONSHIP is missing in the 'defaultValue' prop of either its Controller (https://react-hook-form.com/api#Controller) or useForm (https://react-hook-form.com/api#useForm)`

Ok, think I got that.

**Review by thegrims — APPROVED** [review]

**Pull request merged**